### PR TITLE
Casino game statistics - net total

### LINKED ISSUE #403: Casino game statistics - net total
**Describe the solution you'd like**
Add "Net Total" (Total won - Total lost) to casino statistics.

Torn Forum suggestion: https://www.torn.com/forums.php#/p=threads&f=4&t=16211909&b=0&a=0

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #403 .

### CODE REVIEW

File: torntools/manifest.json [hunk 1]
```diff
@@ -220,6 +220,22 @@
...
+			"run_at": "document_end"
+		},
+		{
+			"matches": ["https://www.torn.com/bookies.php*"],
+			"include_globs" : ["*#/stats/*"],
+			"js": ["scripts/content/loader/ttCasinoStatistics.js"],
+			"run_at": "document_end"
+		},
+		{
+			"matches": ["https://www.torn.com/loader.php?sid=view*"],
```
Comment (DeKleineKobini) [inline torntools/manifest.json:235]: Why add it twice?

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]
You are missing:
- [x] https://www.torn.com/loader.php?sid=rouletteStatistics
- [ ] https://www.torn.com/bookies.php#/stats --> doesn't trigger a page reload it seems

`Uncaught (in promise) TypeError: Cannot read property 'parentElement' of undefined at ttCasinoStatistics.js:16` on:
- [x] https://www.torn.com/loader.php?sid=viewCrapsStats (your stats missing)
- [x] https://www.torn.com/loader.php?sid=viewKenoStats (your stats missing)
- [x] https://www.torn.com/loader.php?sid=viewHighLowStats (your stats missing)
- [x] https://www.torn.com/loader.php?sid=viewBlackjackStats (your stats missing)
- [ ] https://www.torn.com/loader.php?sid=viewPokerStats (~both stats missing~ overall stat missing)
- [ ] https://www.torn.com/loader.php?sid=viewRussianRouletteStats (~both stats missing~ overall stat missing)

File: torntools/manifest.json (hunk 1 above)
Comment (Sashank999) [reply torntools/manifest.json]: That was actually intended for some other page, Fixed.

**Comment by Sashank999** [comment]
> You are missing:
> 
> * https://www.torn.com/loader.php?sid=rouletteStatistics
> * https://www.torn.com/bookies.php#/stats --> doesn't trigger a page reload it seems
> 
> `Uncaught (in promise) TypeError: Cannot read property 'parentElement' of undefined at ttCasinoStatistics.js:16` on:
> 
> * https://www.torn.com/loader.php?sid=viewCrapsStats (your stats missing)
> * https://www.torn.com/loader.php?sid=viewKenoStats (your stats missing)
> * https://www.torn.com/loader.php?sid=viewHighLowStats (your stats missing)
> * https://www.torn.com/loader.php?sid=viewBlackjackStats (your stats missing)
> * https://www.torn.com/loader.php?sid=viewPokerStats (both stats missing)
> * https://www.torn.com/loader.php?sid=viewRussianRouletteStats (both stats missing)

Fixed.

**Comment by DeKleineKobini** [comment]
> Fixed.

Updated my post, still not everything works.

**Comment by Sashank999** [comment]
Russian Roulette and Poker have only "Total money gain" fields so, they don't need "Net Total".
The bookies don't trigger a page reload. The gains and losses don't change when you change the game. TORN doesn't change those, so I didn't add any code to listen them.

**Comment by DeKleineKobini** [comment]
> Russian Roulette and Poker have only "Total money gain" fields so, they don't need "Net Total".
> The bookies don't trigger a page reload. The gains and losses don't change when you change the game. TORN doesn't change those, so I didn't add any code to listen them.

Overlooked the fact on rr and poker, sorry. 
I do think that we should have it work on bookies too though. Might not be as easy, but nobody said that it would be

**Comment by DeKleineKobini** [comment]
I've opened a new PR for this, so I'll be closing this.

**Pull request closed without merging**